doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -4,43 +4,35 @@ High Level Architecture
-The following chapters only describe the specific features required by the
-Anuket Reference Architecture, and how they are expected to be implemented.
-For information related to standard Kubernetes features and capabilities, see
-the standard Kubernetes documentation.
-
-While this reference architecture provides options for pluggable components,
-such as service mesh, the focus of the reference architecture
-is on the abstracted interfaces and features that are required for Telco-type
-workload management and execution.
-
-Chapter 5 of the Reference Model (RM) describes the hardware and software profiles
-that are descriptions of the capabilities and features that the Cloud Infrastructure
-provides to the workloads. Figure 5-3 in the RM (also shown below)
-depicts a high-level view of the software profile features that apply to each
-instance profile (basic and high-performance). For more information on the instance profiles,
-see :ref:`ref_model:chapters/chapter04:profiles`.
+The Anuket Reference Architecture (RA2) for Kubernetes based cloud infrastructure is intended to be an industry
+standard-independent Kubernetes reference architecture that is not tied to any specific offering or distribution. 
+No vendor-specific enhancements are required to achieve conformance with the Anuket specifications.
+Conformance to these specifications can be achieved by using upstream components or features that are developed by the
+open source community and is ensured by successfully running RC2 conformance testing suite.
+
+By using the Reference Architecture (RA2) for Kubernetes based cloud infrastructure specifications, operators can
+deploy infrastructure that will run any VNF or CNF that has successfully run on an RA2-conformant infrastructure. The
+purpose of this chapter is to outline all the components required to provide Telco-grade Kubernetes in a consistent and
+reliable way. The specification of how to setup these components is detailed in 
+:ref:`Chapter 04<chapters/chapter04:component level architecture>`.
+
+Kubernetes is already well-documented and widely deployed as an open source project hosted by the Cloud Native
+Computing Foundation (CNCF). For information related to standard Kubernetes features and capabilities, refer to
+the standard Kubernetes documentation. Full documentation of the Kubernetes code and project can be found on the
+`Kubernetes docs page <https://kubernetes.io/docs/home/>`__.
+The following chapters describe the specific features required by the Anuket Reference Architecture, and how they are
+expected to be implemented.
+
+While this reference architecture provides options for modular components, such as service mesh, the focus of the
+Reference Architecture is on the abstracted interfaces and features that are required for Telco workload management and
+execution.
+
+:doc:`Chapter 4<ref_model:chapters/chapter04>` of the Reference Model (RM) describes the hardware and software profiles
```

Comment:
```suggestion
:ref:`Chapter 4<ref_model:chapters/chapter04>` of the Reference Model (RM) describes the hardware and software profiles
```

Reply:
> Suggested change 
> :doc:`Chapter 4<ref_model:chapters/chapter04>` of the Reference Model (RM) describes the hardware and software profiles
> :ref:`Chapter 4<ref_model:chapters/chapter04>` of the Reference Model (RM) describes the hardware and software profiles

@pgoyal01 thanks - `ref` only works for links to a label within the document, for linking to a whole page (like ch4 in this case), only `doc` works - cf https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-doc-role

Reply:
(we would know for sure if we still had builds for every PR)